### PR TITLE
Feature/checkout icon labels

### LINKED ISSUE #121: Show payment methods icon on label

### PR DESCRIPTION
**Pull request by guilhermepiovesan**
closes #121

### CODE REVIEW

File: app/design/frontend/base/default/template/ebanx/form/baloto.phtml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-baloto.png' ?>" width="40" />
-            </div>
+
+            <?php if(!$showInlineIcon) {
+				echo $icon;
+			} ?>
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/baloto.phtml]: The indentation on lines 17 and 16 aren't the same as the indentation on line 15.

File: app/design/frontend/base/default/template/ebanx/form/boleto.phtml [hunk 2]
```diff
@@ -1,13 +1,20 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-banking-ticket.png" width="40" /></div>';
 ?>
 
-<ul class="form-list ebanx-payment-method ebanx-br-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/boleto.phtml]: Yeah, I know I was the one who told you to do it like this, but I think @guilhermepiovesan was right. Maybe it's nicer to use `<?= $showInlineIcon ? $icon : '' ?>`

File: app/design/frontend/base/default/template/ebanx/form/boleto.phtml [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
+
+<ul class="form-list ebanx-payment-method ebanx-br-method" id="payment_form_<?= $this->getMethodCode() ?>" style="display:none;">
     <li>
-        <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-banking-ticket.png' ?>" width="40" />
-            </div>
+        <div class="<?= $this->getMethodCode() ?>-instructions-content">
+
+            <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/boleto.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/creditcard_br.phtml [hunk 4]
```diff
@@ -1,12 +1,19 @@
 <?php
     $_code = $this->getMethodCode();
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-credit-card-br.png" width="200" /></div>';
 ?>
 
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/creditcard_br.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/creditcard_br.phtml [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+    echo $icon;
+} ?>
+
 <ul class="form-list ebanx-payment-method ebanx-br-method" id="payment_form_<?php echo $_code ?>" style="display:none;">
-    <div>
-        <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-credit-card-br.png' ?>" width="200" />
-    </div>
+
+    <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/creditcard_br.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/creditcard_mx.phtml [hunk 6]
```diff
@@ -1,12 +1,19 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
     $_code = $this->getMethodCode();
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-credit-card-mx.png" width="115" /></div>';
 ?>
 
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/creditcard_mx.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/creditcard_mx.phtml [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
+    echo $icon;
+} ?>
+
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $_code ?>" style="display:none;">
-    <div>
-        <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-credit-card-mx.png' ?>" width="115" />
-    </div>
+
+    <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/creditcard_mx.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/debitcard.phtml [hunk 8]
```diff
@@ -1,11 +1,19 @@
 <?php
     $_code = $this->getMethodCode();
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-debit-card.png" width="141" /></div>';
 ?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/debitcard.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/debitcard.phtml [hunk 9]
```diff
@@ -1,11 +1,19 @@
...
+    echo $icon;
+} ?>
+
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $_code ?>" style="display:none;">
-    <div>
-        <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-debit-card.png' ?>" width="141" />
-    </div>
+
+    <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/debitcard.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/oxxo.phtml [hunk 10]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+	$skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-oxxo.png" width="40" /></div>';
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/oxxo.phtml]: Same as above ⬆️ (not the indentation one)

File: app/design/frontend/base/default/template/ebanx/form/oxxo.phtml [hunk 11]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+	$skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-oxxo.png" width="40" /></div>';
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/oxxo.phtml]: The indentation on lines 3 and 4 are not the same as the indentation on line 2

File: app/design/frontend/base/default/template/ebanx/form/oxxo.phtml [hunk 12]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-oxxo.png' ?>" width="40" />
-            </div>
+
+        	<?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/oxxo.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/pagoefectivo.phtml [hunk 13]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+	$skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-pagoefectivo.png" width="40" /></div>';
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/pagoefectivo.phtml]: Same as above ⬆️ (indentation)

File: app/design/frontend/base/default/template/ebanx/form/pagoefectivo.phtml [hunk 14]
```diff
@@ -1,11 +1,20 @@
...
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/pagoefectivo.phtml]: Same as above ⬆️ (not indentation)

File: app/design/frontend/base/default/template/ebanx/form/pagoefectivo.phtml [hunk 15]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-pagoefectivo.png' ?>" width="40" />
-            </div>
+
+        	<?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/pagoefectivo.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/pse.phtml [hunk 16]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+    $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-eft.png" width="40" /></div>';
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/pse.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/pse.phtml [hunk 17]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-eft.png' ?>" width="40" />
-            </div>
+
+            <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/pse.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/safetypay.phtml [hunk 18]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+    $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-safetypay.png" width="40" /></div>';
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/safetypay.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/safetypay.phtml [hunk 19]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-safetypay.png' ?>" width="40" />
-            </div>
+
+            <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/safetypay.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/sencillito.phtml [hunk 20]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+	$skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-sencillito.png" width="40" /></div>';
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/sencillito.phtml]: Same as above ⬆️ (indentation)

File: app/design/frontend/base/default/template/ebanx/form/sencillito.phtml [hunk 21]
```diff
@@ -1,11 +1,20 @@
...
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/sencillito.phtml]: Same as above ⬆️ (not indentation)

File: app/design/frontend/base/default/template/ebanx/form/sencillito.phtml [hunk 22]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-sencillito.png' ?>" width="40" />
-            </div>
+
+        	<?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/sencillito.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/servipag.phtml [hunk 23]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+	$skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-servipag.png" width="40" /></div>';
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/servipag.phtml]: Same as above ⬆️ (indentation)

File: app/design/frontend/base/default/template/ebanx/form/servipag.phtml [hunk 24]
```diff
@@ -1,11 +1,20 @@
...
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/servipag.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/servipag.phtml [hunk 25]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-servipag.png' ?>" width="40" />
-            </div>
+
+        	<?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/servipag.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/tef.phtml [hunk 26]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+    $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-tef.png" width="115" /></div>';
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/tef.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/tef.phtml [hunk 27]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-br-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-tef.png' ?>" width="115" />
-            </div>
+
+            <?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/tef.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/wallet.phtml [hunk 28]
```diff
@@ -1,11 +1,20 @@
-<?php $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN); ?>
+<?php
+	$skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-account.png" width="70" /></div>';
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/wallet.phtml]: Same as above ⬆️ (indentation)

File: app/design/frontend/base/default/template/ebanx/form/wallet.phtml [hunk 29]
```diff
@@ -1,11 +1,20 @@
...
+?>
+
+<?php if($showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/wallet.phtml]: Same as above ⬆️

File: app/design/frontend/base/default/template/ebanx/form/wallet.phtml [hunk 30]
```diff
@@ -1,11 +1,20 @@
...
+} ?>
 
 <ul class="form-list ebanx-payment-method ebanx-br-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-account.png' ?>" width="70" />
-            </div>
+
+        	<?php if(!$showInlineIcon) {
```
Comment (gpressutto5) [inline app/design/frontend/base/default/template/ebanx/form/wallet.phtml]: Same as above ⬆️

File: app/code/community/Ebanx/Gateway/etc/system.xml [hunk 31]
```diff
@@ -156,6 +156,24 @@
...
+                        <payment_methods_visualization_title translate="label">
+                            <label>Payment Methods Visualization</label>
+                            <frontend_model>adminhtml/system_config_form_field_heading</frontend_model>
+                            <sort_order>170</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                        </payment_methods_visualization_title>
+
+                        <payment_methods_visualization translate="label">
+                            <label>Show payment methods logo for better conversion on your checkout</label>
```
Comment (cezarlz) [inline app/code/community/Ebanx/Gateway/etc/system.xml]: Were these texts revised by Larissa?

File: app/design/frontend/base/default/template/ebanx/form/baloto.phtml [hunk 32]
```diff
@@ -1,13 +1,16 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-baloto.png" width="40" /></div>';
```
Comment (cezarlz) [inline app/design/frontend/base/default/template/ebanx/form/baloto.phtml]: I think we can improve it and put something like:

```php
<?php if ($showInlineIcon): ?>
    <div><img src="<?= $skin ?>frontend/base/default/ebanx/ebanx-baloto.png" width="40" /></div>
<?php endif ?>
```

This way, we can remove some unnecessary variables, IMO.

File: app/design/frontend/base/default/template/ebanx/form/boleto.phtml [hunk 33]
```diff
@@ -1,13 +1,16 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-banking-ticket.png" width="40" /></div>';
```
Comment (cezarlz) [inline app/design/frontend/base/default/template/ebanx/form/boleto.phtml]: Same here.

File: app/design/frontend/base/default/template/ebanx/form/creditcard_br.phtml [hunk 34]
```diff
@@ -1,12 +1,15 @@
 <?php
     $_code = $this->getMethodCode();
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-credit-card-br.png" width="200" /></div>';
 ?>
 
+<?= $showInlineIcon ? $icon : '' ?>
```
Comment (cezarlz) [inline app/design/frontend/base/default/template/ebanx/form/creditcard_br.phtml]: Same here.

File: app/design/frontend/base/default/template/ebanx/form/creditcard_mx.phtml [hunk 35]
```diff
@@ -1,12 +1,15 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
     $_code = $this->getMethodCode();
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
```
Comment (cezarlz) [inline app/design/frontend/base/default/template/ebanx/form/creditcard_mx.phtml]: Same all of them...

File: app/code/community/Ebanx/Gateway/etc/system.xml (hunk 31 above)
Comment (andercampanha) [inline app/code/community/Ebanx/Gateway/etc/system.xml]: She was the one who sent me them and told me to put it there

**Comment by gpressutto5** [comment]
[Config no admin para ícones fora do box](https://trello.com/c/vI7I3C9K/11-config-no-admin-para-%C3%ADcones-fora-do-box)

**Pull request merged**